packages/server/src/app.module.ts

```diff
@@ -32,6 +35,28 @@ import { OrganizationsModule } from './organizations/organizations.module';
     DocumentsModule,
     NftsModule,
     OrganizationsModule,
+    MailerModule.forRoot({
+      transport: {
+        host: 'smtp.sendgrid.net',
+        port: 587,
+        secure: false,
```

Comment:
What does secure=false mean here?

Reply:
It will use STARTTLS. I think we need to set this up in the server somehow. I think this can be a separate ticket IMO and maybe we can expose them as ENV vars? I do not have this set on my machine and I do not need it for testing. Maybe we add all of this values as envs? HOST, PORT, SECURE?

Reply:
So this emails will contain a password recovery phrase or alike, and we should make the connection to the remote server as secure as possible. I would like to test it out with this release, since it is a security concern.

Reply:
So I added this values as a env vars and secure defaults to true.